pages/linux/entr.md

```diff
@@ -0,0 +1,23 @@
+# entr
+
+> Run arbitrary commands when files change.
+
+- Rebuild with `make` if any files in any subfolders change:
+
+`ag -l | entr make`
+
+- Rebuild and test with `make` if `.c` sources in current directory changes:
+
+`ls *.c | entr 'make && make test'`
+
+- Reload the MuPDF viewer whenever the source `.pdf` changes:
```

Comment:
What's MuPDF? I'm unsure as to how this is different to the previous example.

Reply:
It's actually not different, just another example on how to send signals using the command. I've added an updated description below. Alternatively it could be left out. What are your thoughts?
```suggestion
- Reload a PDF viewer (`mupdf`) by sending a `SIGHUP` whenever any `.pdf` sources change in the current folder:
```

Reply:
Yes, I believe it can be left out.